.github/workflows/alternative-ci-runners-3.yml

```diff
@@ -0,0 +1,45 @@
+name: Alternative CI Runners 3
+
+on:
+  # Run the workflow every day TWICE:
+  # 1. 9:06AM UTC (low activity)
+  # 2. 9:26AM UTC (cache test - high chance of no code changes)
+  schedule:
+    - cron: "6,26 9 * * *"
+  # Enable manual trigger for on-demand runs - helps when debugging
+  workflow_dispatch:
+
+permissions:
+  contents: read
+
+jobs:
+  docs-lint-on-magicache:
+    uses: ./.github/workflows/_dagger_on_magicache.yml
+    with:
+      function: docs lint
+      timeout: 20
```

Comment:
While `docs-lint` caches well, it is known to use up to 8CPUs (this instance has 16CPUs):
<img width="1304" alt="image" src="https://github.com/user-attachments/assets/7e7a604b-1374-4fe5-b8ad-d8dba049ceae" />

Only provisioning 4CPUs might make it run artificially slower.

Reply:
Fixed by running on 16c runners **only** - same as all other alternative workflows.